bip-0360.mediawiki

```diff
@@ -177,7 +177,7 @@ While P2TR outputs (and ECC-based key-path spends) will remain an option for fol
 
 First off, P2TSH script-path spends are larger than P2TR key-path spends, because you have to include the control byte and post the script being spent. That said, script-path spends from P2TSH outputs will be slightly smaller than script-path spends from P2TR outputs because there is no need to reveal any internal public key. For a more complete comparison of input witness sizes, the “Transaction Size and Fees” section may be reviewed later in this proposal.
 
-Additionally, there is a privacy tradeoff when comparing P2TSH and P2TR, which is that users will functionally reveal that they are spending from a script tree whenever they are using P2TSH output, since P2TSH outputs can only be spent via the script-path spend. In P2TR when you spend an output as a key-path spend, you don’t reveal any script or if you had any scripts at all. This trade-off only exists when comparing P2TR key-path spends to P2TSH script-path spends; P2TR and P2TSH provide the same level of privacy when both are script-path spends.
+Additionally, there is a privacy tradeoff when comparing P2TSH and P2TR. That is, since P2TSH outputs can only be spent via the script-path spend, users functionally reveal that they are a utilizing a script tree whenever they are using a P2TSH output. And, of course, a user reveals the script policy of their chosen script whenever spending from the P2TSH output. In P2TR when you spend an output as a key-path spend, you don’t reveal any script or if you had any scripts at all.
```

Comment:
`the script policy of their chosen script` is this a typo, I'm not sure what script policy means here.

Reply:
Indeed, 'script policy' is an informal term. Here it means the spending conditions of the script. 

P2TSH outputs are identifiable on the blockchain as such. Then, at spend time the spending conditions of a particular script are revealed and satisfied.